helpers/config.py

```diff
@@ -246,7 +263,7 @@ def frontend(self):
             configuration files
         """
         return not self.multi_servers or \
-               self.__config.get('server_role') == 'frontend'
+            self.__dict['server_role'] == 'frontend'
```

Comment:
Actually it will. 
if  `self.__dict['server_role']` is empty,  it means we are not in the multi servers configuration, so `not self.multi_servers` will be True and the rest of expression would be evaluated. 

Anyway `self.__dict['server_role']` default is back to `frontend`

Reply:
Ok perfect 👌